core/controllers/base_test.py

```diff
@@ -288,6 +293,34 @@ def test_keys_match_en_qqq(self):
         qqq_key_list = self._extract_keys_from_json_file('qqq.json')
         self.assertEqual(en_key_list, qqq_key_list)
 
+    def test_keys_in_source_code_match_en(self):
+        """Tests that keys in HTML files are present in en.json."""
+        en_key_list = self._extract_keys_from_json_file('en.json')
+        dirs_to_search = ['core', 'extensions']
+        files_checked = 0
+        missing_keys_count = 0
+        for directory in dirs_to_search:
+            for root, _, files in os.walk(os.path.join(
+                    os.getcwd(), directory)):
+                for filename in files:
+                    if filename.endswith('.html'):
+                        files_checked += 1
+                        html_key_list = self._extract_keys_from_html_file(
+                            os.path.join(root, filename))
+                        if set(html_key_list) <= set(en_key_list):
```

Comment:
I had to do this because the lint would not allow me to write `if not set(html_key_list) <= set(en_key_list)` suggesting me to change it to `if set(html_key_list) > set(en_key_list)` which is different when dealing with sets.